src/bin/wasmer.rs

```diff
@@ -363,8 +380,13 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
     #[cfg(feature = "backend-llvm")]
     {
         if options.backend == Backend::LLVM {
+            let options = options.backend_llvm_options.clone();
             unsafe {
```

Comment:
If we're going to use mutable global variables, I'm a fan of what I did in Wapm which is wrap it in a write-once abstraction that is thread safe by way of Mutex or RW Lock.  Basically just having the compiler enforce that you can't use it wrong.


I'm generally a fan of no top-level unsafe.  `wasmer.rs` should be business logic and unsafe means you have to care about how things work which slows down reading and modifying the code.  Quickly implementing business logic changes is important from a business and product perspective, so in my opinion they don't mix well.

If this is a temporary fix then it's fine, but I think no unsafe in business logic is something we should stick to.
this looks like this unsafe was already here though -- I wasn't aware we were using this.

Reply:
Ideally these shouldn't be global, they should be tied to each LLVMCompiler object in some fashion.